libbeat/scripts/Makefile

```diff
@@ -87,8 +87,9 @@ SYSTEM_TESTS?=false ## @testing if true, "make test" and "make testsuite" run un
 STRESS_TESTS?=false ## @testing if true, "make test" and "make testsuite" run also run the stress tests
 STRESS_TEST_OPTIONS?=-timeout=20m -race -v
 GOX_OS?=linux darwin windows freebsd netbsd openbsd ## @Building List of all OS to be supported by "make crosscompile".
-GOX_OSARCH?=!darwin/arm !darwin/arm64 ## @building Space separated list of GOOS/GOARCH pairs to build by "make crosscompile".
+GOX_OSARCH?=!darwin/arm !darwin/386 !linux/386 !windows/386 !freebsd/386 !netbsd/386 !openbsd/386 !linux/ppc64 ## @building Space separated list of GOOS/GOARCH pairs to build by "make crosscompile".
```

Comment:
What's the reason for this change?

Reply:
because the latest version of `gox `generates too many different variants of OS/ARCH, and some of them are unsupported by GO.

Reply:
Gotcha, I wonder whether this should be done in a different PR to avoid mixing concerns

Reply:
done, https://github.com/elastic/beats/pull/37870